.github/workflows/validate-action.yaml

```diff
@@ -0,0 +1,15 @@
+name: Validate action
+on: push
+jobs:
+  ci:
+    name: Ensure that the current build output is checked in
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Run build
+      run: cd deploy-basic-service && npm install && npm run package
+    - name: Check if build output matches checked in dist
```

Comment:
I'd prefer to keep `dist` out of `master`, and commit it only when releasing.

Reply:
What would you propose as the standard procedure for doing a release then?

Reply:
The standard release procedure wouldn't be ideal:

```shell
git checkout -b release
npm run build
git add -f dist
git commit 'release 1.2.3'
git tag 1.2.3
git branch -D release
```

The benefit would be that unknowing contributors & automated tools like Greenkeeper/Dependabot wouldn't cause `dist` to drift away from the source code.